mix.exs

```diff
@@ -38,15 +38,14 @@ defmodule Phoenix.GenSocketClient.Mixfile do
       {:poison, "~> 2.0", only: [:dev, :test]},
       {:phoenix, "~> 1.3", only: :test},
       {:cowboy, "~> 1.0", only: :test},
-      {:gproc, "~> 0.5.0", only: :test},
       {:ex_doc, "~> 0.17.1", only: :docs}
     ]
   end
 
   defp elixirc_paths(:test), do: ["lib", "test/support"]
   defp elixirc_paths(_), do: ["lib"]
 
-  defp applications(:test), do: [:logger, :websocket_client, :gproc, :cowboy, :phoenix]
```

Comment:
We previously automatically started `gproc`, should we now automatically start the `PubSub` registry too? Having to manually start it seems a bit finicky.

Reply:
We need it only for tests, so I opted to have that logic in the test helper, instead of having some conditional, per-env logic in `mix.exs`.

Reply:
Ok, sounds good!